### ISSUE TITLE
`Y020` false-positive for `__slots__`

### ISSUE THREAD

**Issue by sobolevn** [PyCQA/flake8-pyi#290]
This code 

```python
class CommandsParser:
    __slots__ = ("commands", "node")
```

produces

```
./stubs/redis/redis/asyncio/parser.pyi:7:18: Y020 Quoted annotations should never be used in stubs
./stubs/redis/redis/asyncio/parser.pyi:7:30: Y020 Quoted annotations should never be used in stubs
```

I don't think it is correct, because this is how `__slots__` should be defined.

Refs https://github.com/python/typeshed/pull/8830

**Comment by AlexWaygood** [comment]
We have 0 `__slots__` definitions in typeshed at the moment, and I'm not sure whether or not it's important to add `__slots__` definitions in stubs. But I agree that this is a bug in flake8-pyi -- we shouldn't error here!

**Comment by sobolevn** [comment]
Yes, I just realized that we have 0 `__slots__` definitions. See https://github.com/python/typeshed/issues/8832 🙂
